example/src/App.tsx

```diff
@@ -38,7 +38,9 @@ export default function App() {
   const ref = React.useRef<MarkdownTextInput>(null);
 
   return (
-    <View style={styles.container}>
+    <ScrollView
+      contentContainerStyle={[styles.container]}
+      style={[styles.content]}>
```

Comment:
Not sure if we need to pass styles inside arrays here. Also we could move this change to a separate PR since it is not related to the RN version bump.
```suggestion
      contentContainerStyle={styles.container}
      style={styles.content}>
```

Reply:
Done, I'm going to prepare a PR in a minute

Reply:
PR: https://github.com/Expensify/react-native-live-markdown/pull/657